frontend/javascripts/oxalis/view/right-menu/abstract_tree_tab_view.js

```diff
@@ -16,6 +16,7 @@ import AbstractTreeRenderer, {
 import window from "libs/window";
 
 type OwnProps = {|
+  // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
eslint got a bit smarter at some spots, but occasionally the smartness also backfired since it didn't catch some usages. the `portalKey` is an extreme example which we use in the golden layout adapter.